metaci/permissions.py

```diff
@@ -0,0 +1,14 @@
+from django.core.exceptions import PermissionDenied, SuspiciousOperation
```

Comment:
Can you put this file in the metaci/api directory since it's for use with Django Rest Framework?